### DISCUSSION TITLE
Regarding Appium Server

### DISCUSSION THREAD

**Discussion by sg12057** [webdriverio/webdriverio#13112] [General]
Hi Team,

Regarding appium server I have one query , if we get any issues for appium server do we have any fixes or patches for resolving that issues or can we contact support team to resolving our issues.

Regards,
Sindusha

**Comment by christian-bromann** [comment]
Hey @sg12057 , WebdriverIO as well as Appium are Open Source projects and don't have something like support teams. If you find a bug, you are welcome to raise them here but also should be the person who helps contributing to the project to resolve them. This is the nature of open source.
